Mathlib/LinearAlgebra/BilinearForm/Basic.lean

```diff
@@ -115,17 +116,15 @@ lemma smul_right_of_tower (r : S) (x y : M) : B x (r • y) = r • B x y := by
 variable {D : BilinForm R M} {D₁ : BilinForm R₁ M₁}
 
 -- TODO: instantiate `FunLike`
+@[deprecated]
 theorem coe_injective : Function.Injective ((fun B x y => B x y) : BilinForm R M → M → M → R) :=
```

Comment:
I think this one is worth keeping, it's about the "right" coercion and we have no `₂` equivalent.

Reply:
Okay, thanks.